Tctl admin cluster (#1158)
temporalio/documentation: docs/php/activities.md

## Local Activity

In Temporal there is a concept of [Local Activity](/docs/concepts/what-is-a-local-activity). To create a local

> Activity you should use `#[LocalActivityInterface]` (instead of `#[ActivityInterface]`):

Reviewer: Revert this change?

Author: It came up as merge conflict, and I seem to have mixed them up.

Reviewer: ```suggestion
Temporal has a concept of [Local Activity](/docs/concepts/what-is-a-local-activity).
To create a Local Activity you should use `#[LocalActivityInterface]` (instead of `#[ActivityInterface]`):
```

Reviewer: ```suggestion
Temporal has a concept of [Local Activity](/docs/concepts/what-is-a-local-activity).
To create a Local Activity you should use `#[LocalActivityInterface]` (instead of `#[ActivityInterface]`):
```
